### PR TITLE
feat: add virtual keyboard

### PR DESCRIPTION
**Pull request by FrantisekMichalSebestyen**
chrome-capture-2023-3-20 (2)

Not tested on the actual device

### CODE REVIEW

**Comment by eronisko** [comment]
I really like this, I'd just make sure the keyboard keys are at least 1cm in size on the target device.

**Comment by FrantisekMichalSebestyen** [comment]
> I really like this, I'd just make sure the keyboard keys are at least 1cm in size on the target device.

The target device should be 1920x1080 and should be ~48.69cm wide.
One pixel therefore should be ~0.024cm (48.69/1920)
The smallest key is ~42px wide and 40px tall on the designated resolution
Therefore, the smallest key should be around 1cm x 0.96 cm

File: src/components/VirtualKeyboard.vue [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  watch: {
+    input(input) {
+      this.keyboard.setInput(input)
+    },
+  },
+}
+</script>
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style scoped></style>
```
Comment (igor-kamil) [inline src/components/VirtualKeyboard.vue:92]: any reason to keep this?

```suggestion
```

**Review by igor-kamil — APPROVED** [review]
👌 thank you 
looks and works very well!

**Pull request merged**